### PR TITLE
Add users

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _data/users.yml [hunk 1]
```diff
@@ -407,6 +415,14 @@
...
   url: https://shopify.engineering/topics/data-science-engineering
   # https://shopify.engineering/faster-trino-query-execution-infrastructure
+- name: SK Telecom
+  anchor: sk_telecom
+  logo: /assets/images/logos/sk-telecom.png
+  logosmall: /assets/images/logos/sk-telecom-small.png
+  testimonial: |
+    TBD
+  urltext:
+  url: https://www.sktelecom.com/
```
Comment (Chaho12) [inline _data/users.yml:425]:
```suggestion
  url: https://www.sktelecom.com/index_en.html
```
How about linking eng version

File: _data/users.yml [hunk 2]
```diff
@@ -281,6 +281,14 @@
     wide array of workloads ranging from ad-hoc to batch queries.
   urltext:
   url: https://www.myntra.com/
+- name: Naver
+  anchor: naver
+  logo: /assets/images/logos/naver.png
+  logosmall: /assets/images/logos/naver-small.png
+  testimonial: |
+    TBD
```
Comment (Chaho12) [inline _data/users.yml:289]: Am not sure about the width of the sentences but here is our testimonial :) 
```suggestion
    Naver Corporation is a global ICT company, providing South Korea’s number one 
    search portal "NAVER". Naver uses Trino for data analytics and business intelligence
    (BI) purposes. Trino allows Naver to query and process massive amounts of data from 
    different sources, such as user logs and clickstream data, in real-time. This enables Naver 
    to deliver more relevant and accurate search results to users based on their preferences.
```

Comment (Chaho12) [reply _data/users.yml:289]: Could it be possible to hold this pr from merging for Naver Corp? Sry for inconvenience

Comment (Chaho12) [reply _data/users.yml:289]: Here is the updated testimonial :) 

```suggestion
Naver Corporation is a global ICT company, providing South Korea’s number one search portal "NAVER". Naver uses Trino for data analytics and business intelligence (BI) purposes. Trino provides data scientists with quick access to their data lake, via a standard SQL interface. Trino process massive amounts of data from different sources, such as user logs and clickstream data, in real-time.
```

**Comment by bitsondatadev** [comment]
https://github.com/trinodb/trino.io/pull/435

**Pull request closed without merging**